### PR TITLE
Databaseconf

### PR DESCRIPTION
**Pull request by kshitij01042002**
Instructions on how to setup database on your local machine added.
If possible some one try at your end and confirm if there's any other steps needed to be mentioned
#102

### CODE REVIEW

File: DATABASEconf.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+Project uses Google's Firebase to store data and information.
+
+For setting up database on your local machine follow below mentioned steps
+
+- Go to [Firebase console](https://console.firebase.google.com/u/0/?utm_source=firebase.google.com&utm_medium=referral) start a new project
+- Go to build-> Authentication, Keep sign-in method as email and password
```
Comment (narayansoni) [inline DATABASEconf.md:7]: After starting a new project there's need to create/register the web app and while registering we can get all the configurations we want. But we have to enable them as mentioned in line 7 and 8

Comment (kshitij01042002) [reply DATABASEconf.md:7]: Soo what steps should I add in it?

Comment (kshitij01042002) [reply DATABASEconf.md:7]: if we copy the credentails beofre starting the service, the database key is not added in the config, that's why I mentioned to access the configurations(API KEY) in later stage

Comment (narayan954) [reply DATABASEconf.md:7]: > Soo what steps should I add in it?

add creating/registering app step in between line 6 and line 7

Comment (narayan954) [reply DATABASEconf.md:7]: > if we copy the credentails beofre starting the service, the database key is not added in the config, that's why I mentioned to access the configurations(API KEY) in later stage

Yes, that's okay

Comment (narayan954) [reply DATABASEconf.md:7]: > if we copy the credentails beofre starting the service, the database key is not added in the config, that's why I mentioned to access the configurations(API KEY) in later stage

Yes, that's okay

File: DATABASEconf.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Again go in build->Realtime Database click on **Get Started** .
```
Comment (narayan954) [inline DATABASEconf.md:8]: I am not sure but is realtime database required at all?

Comment (narayan954) [inline DATABASEconf.md:8]: I am not sure but is realtime database required at all?

**Review by narayan954 — APPROVED** [review]
lgtm :)

**Pull request merged**